src/sourcelink/eng/Versions.props

```diff
@@ -12,9 +12,8 @@
     <!-- commandline -->
     <SystemCommandLineVersion>2.0.0-beta5.25208.1</SystemCommandLineVersion>
     <!-- msbuild -->
-    <MicrosoftBuildVersion>17.8.3</MicrosoftBuildVersion>
-    <MicrosoftBuildTasksCoreVersion>17.8.3</MicrosoftBuildTasksCoreVersion>
-    <MicrosoftIORedistVersion>6.0.1</MicrosoftIORedistVersion>
+    <MicrosoftBuildVersion>17.11.31</MicrosoftBuildVersion>
```

Comment:
As a standalone package sourcelink might still want to target down to 17.8? Not sure.

Reply:
The msbuild packages are consumed at compile time only in sourcelink, not at run time. It shouldn't matter which version is referenced given that msbuild started pinning the assembly version years years ago. That's at least my understanding, please correct me if I'm wrong here.

Reply:
If you reference a higher package version you can reference APIs that don't exist at runtime when running on a lower MSBuild. This is not likely to be a problem for this version range and usage though, so almost certainly not a big deal.